### DISCUSSION TITLE
DataTable: Allow duplicates key in header with different title

### DISCUSSION THREAD

**Discussion by mrbu-solutions** [carbon-design-system/carbon-components-svelte#1789] [Ideas]
Right now, DataTable won't allow duplicate key names in the header. I've got an error when trying:

![image](https://github.com/carbon-design-system/carbon-components-svelte/assets/53817867/a4666324-0b83-4a95-9321-840a09738bf7)

In some scenario, if I have a timestamp field data and wants to display at one column as Date only and the other column as Time only, I can't accomplish such.

**Comment by metonym** [comment]
The display values don't have to be unique.

```svelte
<DataTable
  headers={[
    { key: "date", value: "Time" },
    { key: "time", value: "Time" },
  ]}
/>
```
